Formula/petsc.rb

```diff
@@ -0,0 +1,40 @@
+class Petsc < Formula
+  desc "Tools for solving partial differential equations in parallel"
```

Comment:
@jedbrown I've rephrased this, do you think it correctly distills the essence of the software suite?

Reply:
If you want a one-liner, why not use the name? Portable, Extensible Toolkit for Scientific Computation

Reply:
@tschoonj thoughts?

Reply:
Proceeding with @jedbrown's suggestion.